app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,10 @@
         android:required="false" />
 
     <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
```

Comment:
What is this permission used for?

Reply:
leftover from broadcaster receiver. Removed.